app/src/main/java/io/homeassistant/companion/android/vehicle/MainVehicleScreen.kt

```diff
@@ -70,27 +72,30 @@ class MainVehicleScreen(
         )
         val SUPPORTED_DOMAINS = SUPPORTED_DOMAINS_WITH_STRING.keys
 
-        private val MAP_DOMAINS = listOf(
+        val MAP_DOMAINS = listOf(
             "device_tracker",
             "person",
             "sensor",
             "zone"
         )
     }
 
+    private var favoriteEntites: Flow<List<Entity<*>>> = flowOf()
```

Comment:
Typo: `Entites`

Use `flowOf<List<Entity<*>>>()` like the other functions, it looks nicer